tests/rptest/tests/schema_registry_test.py

```diff
@@ -141,6 +141,24 @@ def accept_response(resp):
     def _base_uri(self):
         return f"http://{self.redpanda.nodes[0].account.hostname}:8081"
 
+    def _get_kafka_cli_tools(self):
+        sasl_enabled = self.redpanda.sasl_enabled()
+        cfg = self.redpanda.security_config() if sasl_enabled else {}
+        return KafkaCliTools(self.redpanda,
+                             user=cfg.get('sasl_plain_username'),
+                             passwd=cfg.get('sasl_plain_password'))
+
+    def _get_serde_client(self, schema_type: SchemaType, topic: str):
```

Comment:
nit: `_get_serde_client` and `_get_kafka_client` seem identical between SR and PP tests. Should we put them in a shared location? The wrappers for `SchemaRegistryEndpoints` and `PandaProxyEndpoints` could go in there as well.